configure.ac

```diff
@@ -432,6 +433,13 @@ AC_MSG_RESULT($enable_strict)
 
 AC_SUBST(REAUTHORIZE_LIBS)
 
+KEY_AUTH=0
+PKG_CHECK_MODULES(LIBSSH_WITH_AGENT_FD, [libssh >= $LIBSSH_KEYAUTH_VERSION],
```

Comment:
I would suggest checking for the required function instead. That's more idiomatic for autotools.